### PR TITLE
landing page: change export selection to dropdown

### LINKED ISSUE #1046: landing page: change export selection to dropdown
Screenshot 2021-08-04 at 09 35 54

### PR DESCRIPTION
**Pull request by jennur**
closes #1046 

Screenshot 2021-08-04 at 11 27 05

### CODE REVIEW

File: invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/side_bar.html [hunk 1]
```diff
@@ -35,22 +34,28 @@
...
-  </hr>
-  <dd class="top-bottom-padded">
-    <div class="ui celled horizontal list separated-list">
+  <h4>{{ _('Export')}}</h4>
+  <hr class="thin-line"/>
+
+  <div class="selection-wrap">
+    <select class="ui selection dropdown export">
       {# dynamically create the list of export formats #}
+      {%- set ns = namespace (initial_val = "") %}
```
Comment (fenekku) [inline invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/side_bar.html:43]: minor:
```suggestion
      {%- set ns = namespace(initial_val="") %}
```
Didn't know about this. That `namespace` is actually pretty nice.

File: invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/side_bar.html [hunk 2]
```diff
@@ -35,22 +34,28 @@
...
           {%- set export_url = url_for('invenio_app_rdm_records.record_export', pid_value=record.id, export_format=fmt, preview=1) -%}
         {% else %}
           {%- set export_url = url_for('invenio_app_rdm_records.record_export', pid_value=record.id, export_format=fmt) -%}
         {% endif %}
-         <div class="item"><a href="{{ export_url }}">{{ name }}</a></div>
+
+        {% if loop.first %}  
+          {%- set ns.initial_val = export_url %}
+        {% endif %}
+         <option value="{{export_url}}">{{name}} </div>
```
Comment (fenekku) [inline invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/side_bar.html:55]:
```suggestion
         <option value="{{export_url}}">{{name}}</option>
```

**Review by fenekku — APPROVED** [review]
Minor things, but looks nice!

**Comment by jennur** [comment]
This PR/branch is included in #1099

**Comment by lnielsen** [comment]
#1099

**Pull request closed without merging**